rest/route/middleware.go

```diff
@@ -432,8 +432,8 @@ func (m *CommitQueueItemOwnerMiddleware) ServeHTTP(rw http.ResponseWriter, r *ht
 		pr, err := m.sc.GetGitHubPR(ctx, projRef.Owner, projRef.Repo, itemInt)
 		if err != nil {
 			gimlet.WriteResponse(rw, gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
-				StatusCode: http.StatusInternalServerError,
-				Message:    "can't get information about PR",
+				StatusCode: http.StatusBadRequest,
+				Message:    fmt.Sprintf("unable to get pull request info, PR number may be invalid: %s", err),
```

Comment:
Would it be helpful to return the PR number in this case?

Reply:
yep done